src/app/resources/assort/page.tsx

```diff
@@ -1,11 +1,130 @@
+        justify="between"
+      >
+        <SortingControl
+          title={data[0].title}
+          description={data[0].description}
+          src="/images/resources/Vector.png"
+        />
+        <Separator
+          orientation="vertical"
+          className="h-auto self-stretch hidden md:block"
+        />
+        <SortingControl
+          title={data[1].title}
+          description={data[1].description}
+          src="/images/resources/Vector (1).png"
+        />
+      </Flex>
+      <Separator
+        orientation="horizontal"
+        className="w-auto mx-16 my-3 lg:block hidden"
+      />
+      <Flex
+        mx={{ md: "9", initial: "3" }}
+        gap="3"
+        className="text-navy-800"
+        direction="column"
+        align="center"
+        my="8"
+      >
+        <Text size="5" align="center">
+          ASSORT leads to faster, cheaper, and more responsive aid. By providing
+          a “common language” for aid donations, organisations receiving aid can
+          know ahead of time what is on an aid shipment.
+        </Text>
+        <Image
+          src="/images/resources/Vector (2).png"
+          alt="ASSORT"
+          width={22}
+          height={38}
+        />
+        <Text size="4" align="center">
+          When the shipment arrives, they process it much faster than they
+          otherwise could. Indeed, a pilot shipment showed that using ASSORT
+          standards significantly cut down on the receiving organisation’s
+          sorting and processing time. Saving time and volunteer capacity makes
+          a big difference for grassroots aid organisations.
+        </Text>
+      </Flex>
+      <Flex
+        mx={{ md: "9", initial: "2" }}
+        gap={{ initial: "4", md: "0" }}
+        className="text-navy-800"
+        direction={{ md: "row", initial: "column" }}
+        align="center"
+        my="8"
+      >
+        <Guide />
+        <Image
+          src="/images/resources/unsplash_vZJdYl5JVXY.png"
+          alt="ASSORT"
+          width={548}
+          height={282}
+        />
+      </Flex>
+      <Heading
+        align="left"
+        size="6"
+        className="text-navy-900"
+        mx={{ md: "9", initial: "3" }}
+      >
+        ASSORT was established as part of a joint partnership between
+        <ResourceRedirect text="Boxtribute" link="boxtribute.ord" />,{" "}
+        <ResourceRedirect link="distributeaid.org" text="Distribute Aid" />,{" "}
+        <ResourceRedirect link="hermine.global" text="HERMINE" />, and the{" "}
+        <ResourceRedirect text="InterEuropean Human Aid" link="iha.help" />{" "}
+        Association.
+      </Heading>
+      <Flex
+        gap={{ initial: "4", md: "8" }}
+        direction="column"
+        align="center"
+        className="bg-red-50"
+        mt={{ md: "9", initial: "7" }}
+        py="43px"
+        px={{ initial: "3", md: "9" }}
+      >
+        <ResourcesFooter />
+        <Text>
```

Comment:
**Suggestion:** Let's take out this paragraph.  We're not really maintaining the "stay up to date" mailing list anymore.